src/test/java/org/opensearch/security/ssl/OpenSSLTest.java

```diff
@@ -1,247 +0,0 @@
-/*
```

Comment:
Why was this file deleted with this PR? Was it intentional?

Reply:
These tests don't work on my windows machine, using CI to find the other impacted features/tests.  This will be added back before this is ready for PR or I'll have a good reason to keep it removed

Reply:
This has been resolved - please re-review around here